src/components/SideMenu/index.tsx

```diff
@@ -0,0 +1,44 @@
+import { useState } from 'react'
+import { Link } from 'react-router-dom'
+import classnames from 'classnames'
+
+import styles from './SideMenu.module.scss'
+
+function SideMenu() {
+  const [isOpen, setIsOpen] = useState(false)
+
+  const toggleMenu = () => {
+    setIsOpen((current) => !current)
+  }
+
+  return (
+    <>
+      <button
+        className={classnames(
+          styles.hamburger,
+          isOpen ? styles.hamburgerActive : ''
```

Comment:
isOpen && styles.hamburgerActive wont work for classnames? does it omit all falsy values?